tokamak/1-no-currents/BOUT.inp

```diff
@@ -0,0 +1,309 @@
+
+#######################
+# Output variables
+output_ddt = false # Output time derivatives
+verbose = true
+
+#######################
+# Numerical dissipation
+numdiff = -0.01    # Numerical diffusion
+hyper = -1      # Hyper-diffusion
+ExBdiff = -1e-3  # Dissipation in ExB advection terms
+y_hyper_viscos = -1
+z_hyper_viscos = -0.01
+ne_hyper_z = -0.01
+pe_hyper_z = -0.01
+
+vepsi_dissipation = true  # Parallel dissipation on Ve-Vi
+vort_dissipation = false
+
+radial_buffers = true
+radial_inner_width = 4
+radial_outer_width = -1
+radial_buffer_D = 1
+
+# Flux limiters
+kappa_limit_alpha = 0.2 # SOLPS style heat flux limiter
+eta_limit_alpha   = 0.5 # SOLPS style viscosity limiter
+
+#######################
+# Electric field and Ohm's law
+electromagnetic = true  # Electromagnetic? Otherwise electrostatic
+FiniteElMass    = true  # Finite electron mass?
+
+# Electrostatic potential
+split_n0 = false  # Solve n=0 separately?
+split_n0_psi = false
+
+# NOTE: all currents switched off for fluid run
+j_diamag = false
+j_par = false
+
+pe_par         = true  # Parallel pressure gradient: Pe <-> Psi
+resistivity    = true  # Resistivity: Psi -> Pe
+thermal_flux   = true
+thermal_force  = true
+electron_viscosity = false
+ion_viscosity = false
+ion_viscosity_par = true
+thermal_conduction = true
+poloidal_flows = false   # Include ExB flows in the X-Y plane?
+ion_velocity = true      # Evolve ion parallel velocity?
+boussinesq = true   # Use Boussinesq approximation
+
+frecycle = 0.99
+
+carbon_fraction = 0.01
+
+excitation = true # Hydrogen neutral excitation radiation
+
+## Settings for 2D parallel closures
+sinks = false
+sink_invlpar =  0.2  # 5m parallel connection length
+sheath_closure = false
+drift_wave = false
+
+
+#######################
+# Transport coefficients
+classical_diffusion = true  # Collisional diffusion
+
+anomalous_D = 1
+anomalous_chi = 0.5 # Anomalous thermal diffusion [m^2/s]
+anomalous_nu = 0.01   # Anomalous viscosity
+
+ion_neutral = 0.0
+neutral_friction = false  # Friction between plasma and neutrals
+
+# Radial boundary fluxes
+ne_bndry_flux = true
+pe_bndry_flux = true
+vort_bndry_flux = false
+
+ramp_mesh      = false
+ramp_timescale = 1e4
+
+#######################
+# Plasma sheath
+sheath_model = 2  # 0 = Bohn, 1 = Loizu, 2 = Bohm + free density
+sheath_yup = true
+sheath_ydown = true
+sheath_gamma_e = 4   # Electron sheath heat transmission
+sheath_gamma_i = 2.5 # Ion sheath heat transmission
+neutral_gamma = 0.0
+
+startprofiles = true
+
+core_sources = true # Only sources in the core
+adapt_source_p = false   # Feedback on profiles (PI controller)
+adapt_source_n = true
+energy_source = true   # Constant power per particle (at fixed x). False -> Constant power per volume
+source_p = 1e-2       # Rate based on current error (1/timescale)
+source_i = 1e-6       # Integral of error (1/timescale^2)
+source_vary_g11 = true  # Vary source in poloidal angle to better match radial transport
+
+staggered = false
```

Comment:
I can see the setting exists, but looks like it doesn't do anything? Maybe clearer for new users to just delete (here and/or `bool staggered` in the source code) so it doesn't look like something that might be worth changing?